templates/web/src/index.ts.twig

```diff
@@ -9,8 +9,9 @@ export { Client, Query, {{spec.title | caseUcfirst}}Exception } from './client';
 {% for service in spec.services %}
 export { {{service.name | caseUcfirst}} } from './services/{{service.name | caseDash}}';
 {% endfor %}
-export type { Models, Payload, RealtimeResponseEvent, UploadProgress } from './client';
```

Comment:
We could export `Params` here instead. Not sure why it was exported in the first place, but seems reasonable to keep it, just under new name